src/libraries/System.Private.CoreLib/src/System/Collections/Generic/ArraySortHelper.cs

```diff
@@ -7,6 +7,8 @@
 using System.Runtime.InteropServices;
 using Internal.Runtime.CompilerServices;
 
+#pragma warning disable CA1822 // Workaround for analyzer bug
```

Comment:
maybe add the bug ID here so this can be removed when it's fixed?

Reply:
What is the bug?

Reply:
I didn't have time to extract the code into bug yet